### ISSUE TITLE
Support for custom templates

### ISSUE THREAD

**Issue by dpq** [grandrew/plotti.co#6]
I propose to introduce support for custom SVG templates that can have varying layouts. To substitute the required template just change the filename path in the URI, i.e. writing `http://plotti.co/<hash>/3dplot.svg` instead of `http://plotti.co/<hash>/plot.svg` in the object `src` parameter will render the very same data but pull it into `3dplot.svg` template taken from the gallery.

**Comment by grandrew** [comment]
Great idea! Just need an interface for the gallery to submit SVG images. Actually, any type of graphics can be submitted this way - like pie charts, etc.

**Comment by grandrew** [comment]
Actually, as we have sources here - no other interface than github pull-request is required, just the internal ability to load templates, automatic review and merge...

**Comment by dpq** [comment]
Right. So we only need to implement the svg file selector based on the URI.

**Comment by dpq** [comment]
Closed by 07326ae
